### PR TITLE
Feature/dp fixing parallel processing

### PR DESCRIPTION
**Pull request by dav009**
Testing this on the big wiki.
- calling `-action export` will use the regular jsonpedia
- calling `-action export-parallel` will use the full powaar!
- deactivated the benchmark classes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,14 @@ Note that the output path corresponds to a path within the docker container. In
 4. Uncompress the Wikipedia Dump
 5. do:
 
-	SPARKFOLDER/bin/spark-submit --driver-memory 10G --class it.cnr.isti.hpc.wikipedia.cli.MediawikiToJsonCLI json-wikipedia-1.0.0-jar-with-dependencies.jar -input <PATHTODBPEDIADUMP> -output <PATHTONEWJSONPEDIA> -lang <LANG>
+	SPARKFOLDER/bin/spark-submit --driver-memory 10G --class it.cnr.isti.hpc.wikipedia.cli.MediawikiToJsonCLI json-wikipedia-1.0.0-jar-with-dependencies.jar -input <PATHTODBPEDIADUMP> -output <PATHTONEWJSONPEDIA> -lang <LANG> -action export-parallel
```
Comment (Lugrin) [inline README.md]: For some reason this line is not displayed as "code" in github. It may be because of the leading <tab>.

**Comment by Lugrin** [comment]
There are lots of lines that are commented out. Could we have a boolean variable that switches the progress logging on/off?

**Comment by dav009** [comment]
thats not switching the progress on/off but a component which benmarkches the parser.
I might as well remove it, I think we have never really used it ,and I want  it out as it blocks the parallel version

**Comment by Lugrin** [comment]
I think we should either put an on/off switch for the benchmark or remove it altogether. We can do it later if you don't know yet which way is better.

File: README.md (hunk 1 above)
Comment (dav009) [inline README.md]: I think it is shown as intended.

But I can make it look like a code block if you think it looks better

**Comment by dav009** [comment]
I will sack it off then

**Comment by keynmol** [comment]
Important issue: it should be "powaa-ah", not "powaar".

**Comment by keynmol** [comment]
Probably fine.

**Pull request merged**